pandas/core/frame.py

```diff
@@ -1841,7 +1842,7 @@ def info(self, verbose=None, buf=None, max_cols=None, memory_usage=None,
             - If False, never show counts.
 
         """
-        from pandas.io.formats.format import _put_lines
+        from pandas.io.formats.format import buffer_put_lines
```

Comment:
you can reference this directly as ``fmt.buffer_put_lines`` as fmt is already imported